Babylon/commands/macro/destroy.py

```diff
@@ -83,6 +85,15 @@ def destroy(state: dict, azure_token: str, state_to_destroy: pathlib.Path):
     env.store_state_in_local(state=state)
     env.store_state_in_cloud(state=state)
 
+    # deleting web app
+    webapp_id = state['services']['webapp']['webapp_name']
+    logger.info(f"Deleting webapp {webapp_id} ....")
+    azure_token = get_azure_token()
+    subscription_id = state['services']['azure']["subscription_id"]
```

Comment:
we can delete this variable :slightly_smiling_face:

Reply:
Which variable ? :)

Reply:
subscription_id, we don't need it here because ASWservice can access it directly in the state :slightly_smiling_face: